direct/common/subsample.py

```diff
@@ -28,9 +28,12 @@
 
 __all__ = (
     "CalgaryCampinasMaskFunc",
-    "FastMRIRandomMaskFunc",
+    "CartesianEquispacedMaskFunc",
+    "CartesianMagicMaskFunc",
+    "CartesianRandomMaskFunc",
     "FastMRIEquispacedMaskFunc",
     "FastMRIMagicMaskFunc",
+    "FastMRIRandomMaskFunc",
     "Gaussian1DMaskFunc",
     "Gaussian2DMaskFunc",
     "RadialMaskFunc",
```

Comment:
Maybe at some point we should give examples of these masks in our documentation as well with some references.

Reply:
Can you make it an issue so we won't forget?